frontend/src/hooks/useAuth.ts

```diff
@@ -43,6 +43,10 @@ const useAuth = () => {
         errDetail = err.message
       }
 
+      if (Array.isArray(errDetail)) {
+        errDetail = "Something went wrong"
```

Comment:
This is usually when getting a pydantic validation error.

It would be nice to make the error to each field, but it's not straightforward 😊